.github/workflows/gating.yaml

```diff
@@ -0,0 +1,113 @@
+name: Gating
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install required packages
+      run: |
+        sudo apt-get install -y \
+          libkrb5-dev
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox tox-gh-actions
+    - name: Test with tox
+      run: tox
+    - name: Run coveralls-python
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        COVERALLS_FLAG_NAME: python-${{ matrix.python-version }}
+        COVERALLS_PARALLEL: true
+      run: |
+        pip3 install --upgrade pip
+        pip3 install --upgrade setuptools
+        pip3 install --upgrade coveralls
+        coveralls --service=github
+
+  coveralls-finish:
+    name: Finish coveralls-python
+    needs: tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Finished
+        run: |
+          pip3 install --upgrade pip
+          pip3 install --upgrade setuptools
+          pip3 install --upgrade coveralls
+          coveralls --finish --service=github
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  flake8:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
Either flake8 or black just needs to run with one Python version. Can this matrix be removed? Or, what requires this matrix setting?

Reply:
I can remove matrix, I kept it as matrix in case that we would need run flake/black on multiple versions.
With black it seems not needed at all, and with flake given that there is no py2, probably neither. I'll remove.